packages/atomic/src/locales.json

```diff
@@ -1075,8 +1075,8 @@
     "fr": "Visualiser les résultats"
   },
   "organization-is-paused": {
-    "en": "Your Coveo organization is paused due to inactivity and search is currently unavailable.",
-    "fr": "Votre organisation Coveo est en pause pour cause d'inactivité et la recherche est présentement non disponible."
+    "en": "Your organization {{org}} is paused due to inactivity and search is currently unavailable.",
+    "fr": "Votre organisation {{org}} est en pause pour cause d'inactivité et la recherche est présentement non disponible."
```

Comment:
Does sending a request begin the process of restarting the org? If so, I think it would be helpful to include an explanation in the message, e.g.

"Your organization {{org}} was paused due to inactivity, and is currently restarting. Please retry your search in a few minutes."

Reply:
Description:

```
 "organization-will-resume": {
    "en": "Your organization is resuming and will be available shortly.",
    "fr": "Votre organisation redeviendra active et sera bientôt disponible."
  },
```